docs/units-and-global-variables.rst

```diff
@@ -111,6 +111,21 @@ same as ``sha3(uint16(0x12))``.
 
 It might be that you run into Out-of-Gas for ``sha256``, ``ripemd160`` or ``ecrecover`` on a *private blockchain*. The reason for this is that those are implemented as so-called precompiled contracts and these contracts only really exist after they received the first message (although their contract code is hardcoded). Messages to non-existing contracts are more expensive and thus the execution runs into an Out-of-Gas error. A workaround for this problem is to first send e.g. 1 Wei to each of the contracts before you use them in your actual contracts. This is not an issue on the official or test net.
 
+.. index:: send, balance
+
+Address Related
+---------------
+
+``<address>.balance`` (``uint256``):
```

Comment:
Please add links to the address type here and from the address type to here, so that we do not forget updating both if something changes.